appcenter-crashes/Crashes.d.ts

```diff
@@ -0,0 +1,42 @@
+export const enum UserConfirmation {
+  DONT_SEND = 0,
+  SEND = 1,
+  ALWAYS_SEND = 2
+}
+
+export interface ListenerMap {
+  onBeforeSending?: () => void;
+  onSendingSucceeded?: () => void;
+  onSendingFailed?: () => void;
+  getErrorAttachments?: (report) => Promise<[TextAttachment, FileAttachment]>;
+}
+
+interface TextAttachment {
```

Comment:
Text and File attachment are the same ErrorAttachmentLog class on android and .net SDKs, please stick to other SDK conventions for now. The methods are actually statics that return the same type.